libs/core/tests/unit_tests/prompts/test_structured.py

```diff
@@ -51,7 +51,7 @@ class OutputSchema(BaseModel):
 
     chain = prompt | model
 
-    assert chain.invoke({"hello": "there"}) == OutputSchema(name="yo", value=42)
+    assert chain.invoke({"hello": "there"}) == OutputSchema(name="yo", value=42)  # type: ignore[comparison-overlap]
```

Comment:
chain is a `Runnable[dict, BaseMessage]` so the invoke return should normally be `BaseMessage`.
I'm not sure if something can be done easily when a structured output is used.